README.org

```diff
@@ -14,7 +14,7 @@
   - [[#acknowledgements][Acknowledgements]]
 
 ** Introduction
-   Gkroam is a lightweight [[https://roamresearch.com][Roam Research]] replica, built on top of emacs org-mode. It uses 'ripgrep' to search pages' links and insert references at the bottom of org pages automatically. Almostly, gkroam imitates roam research in as many aspects as possible.
```

Comment:
@Kinneyzhang , I have not heard the word "Almostly" used before. The sentence sounded odd with it being used.